pkgs/test_api/lib/src/backend/declarer.dart

```diff
@@ -91,6 +91,11 @@ class Declarer {
   /// Whether any tests and/or groups have been flagged as solo.
   bool get _solo => _soloEntries.isNotEmpty;
 
+  /// A full test name to match when running in single test mode.
+  ///
+  /// Groups which are not a strict prefix of this name will be ignored.
+  final String? _filterToSingleTestName;
```

Comment:
This sounds like a boolean to me. I would just call it `_testName`, or possibly `_singleTestName`.

Reply:
What do you think of `_exactTestName` or `_fullTestName`? I realized that `single` isn't quite appropriate here because `Declarer` doesn't do anything to enforce that there is exactly one. That enforcement happens in `directRunSingleTest`.

Reply:
`_fullTestName` SGTM